src/snowcli/cli/snowpark_shared.py

```diff
@@ -104,10 +103,6 @@ def snowpark_package(
                     split_requirements.snowflake
                 ):
                     f.write(package.line + "\n")
-        if pack_dir:
-            utils.recursive_zip_packages_dir(pack_dir, "app.zip")
-        else:
-            utils.standard_zip_dir("app.zip")
-    else:
-        utils.standard_zip_dir("app.zip")
+
+    zip_current_dir("app.zip")
```

Comment:
```suggestion
    zip_current_dir(dest_zip="app.zip")
```
Just for readability sake

Reply:
Done